hack/observability/capi-state-metrics/kube-state-metrics/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for kube-state-metrics.
+prometheusScrape: true
+image:
+  # repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
+  # tag: v2.5.0
+  # custom image which includes changes > v2.5.0 required for custom resource metrics
+  repository: chrischdi/kube-state-metrics
+  tag: v2.5.0-c766f5cb
```

Comment:
TODO: update as soon as new release is out

Reply:
exactly

Reply:
moved over to https://github.com/kubernetes-sigs/cluster-api/pull/7095/files#r953778582